setup.py

```diff
@@ -8,7 +8,7 @@
     'moto==1.3.16',
     'mock==2.0.0',
     'pytest==3.10.1',
-    'flake8==3.8.4'
+    'flake8==4.0.0'
```

Comment:
We need to add a comment here so future us knows why this is pinned to this specific version.

```suggestion
    # NOTE: this is pinned to 4.0.0 so as to pin importlib to a version that works.
    'flake8==4.0.0'
```